leapp/cli/upgrade/__init__.py

```diff
@@ -182,6 +185,7 @@ def process_whitelist_experimental(repositories, workflow, configuration, logger
 @command_opt('whitelist-experimental', action='append', metavar='ActorName', help='Enables experimental actors')
 @command_opt('debug', is_flag=True, help='Enable debug mode', inherit=False)
 @command_opt('verbose', is_flag=True, help='Enable verbose logging', inherit=False)
+@command_opt('no-rhsm', is_flag=True, help='Skip Red Hat Subscription Manager checks', inherit=False)
```

Comment:
It's something that customers could want to use and skipping checks is not exactly what about this is. What about something like:

```suggestion
@command_opt('no-rhsm', is_flag=True, help='Use only custom repositories and skip actions with Red Hat Subscription Manager', inherit=False)
```

You can modify it. Just take the point.